### ISSUE TITLE
Migrating from a Simple CNN Architecture to AdderNet

### ISSUE THREAD

**Issue by aliarastehfard** [huawei-noah/AdderNet#71]
Hi. I have a CNN architecture that I trained on CIFAR-10 with and without AdderNet. I could reach an accuracy of over %80 without AdderNet but when I used AdderNet it got stuck at %10 accuracy. Is there anything wrong with my implementation? All I did was to replace `nn.conv2D` with `adder.adder2d`. Isn't it supposed to work like this? How do you suggest I should migrate from a simple CNN architecture to AdderNet? Thank you!

**Comment by aliarastehfard** [comment]
During the train_test the accuracy stayed the same at %10 for 100 epochs.

**Comment by aliarastehfard** [comment]
The problem was that I wasn't using Batch Normalization which we were instructed to use in the paper,
